.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,20 +1,45 @@
 ---
 name: Feature request
-about: Suggest an idea for this project
+about: Share an idea
 title: ''
-labels: New Feature
+labels: language feature request
 assignees: ''
 
 ---
 
-**Is your feature request related to a problem? Please describe.**
-A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+<!--
+This repository tracks issues and requests related to the implementation of the
+Swift compiler, standard library, runtime, and tools that provide IDE support
+for Swift (e.g. code completion). If your feature request relates to the
+implementation of a proprietary (closed-source) Apple framework such as UIKit,
+SwiftUI, Combine, etc., please report it to https://feedbackassistant.apple.com
+instead.
+-->
 
-**Describe the solution you'd like**
-A clear and concise description of what you want to happen.
+**Motivation**
+<!-- Describe the problem that your feature seeks to address. -->
 
-**Describe alternatives you've considered**
-A clear and concise description of any alternative solutions or features you've considered.
+**Solution**
+<!--
+Describe your solution to the problem. Provide examples and describe how
+they work.
+-->
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+**Alternatives considered**
+<!--
+Describe any alternative approaches or features that you have considered in
+addressing the problem, and why you chose this approach instead.
+-->
+
+<!--
+Add any other context about your feature as appropriate. For example, link out
+to a discussion on the Swift forums (https://forums.swift.org).
+
+If you haven't found or started a thread yet, consider socializing your idea
```

Comment:
I think this whole section needs to be lifted to the very top because this is the main consideration we'd like for people to have. WDYT?

Reply:
Yeah, I just don’t want it to sound like a substitute for submitting the issue, or that it should be done at all.

Reply:
Hmm. So you think that sharing the idea on the forums is something to consider before, not after creating a tracking issue?

Reply:
Do you mind if I explicitly mention to consider it in addition to submitting the request? I’d be nice to be able to navigate through ideas/proposals — including ones that met a resolution before or without becoming an official proposal — in our issues database, not just the forums.

Reply:
Sure, that would be fine too, we just need to put it some where visible that's all.